Test.java

```diff
@@ -0,0 +1,164 @@
+
+public class Test {
+
+    static Team galatasaray = new Team(1, "galatasaray", "GS");
+    static Team fenerbahce = new Team(2, "fenerbahce", "FB");
+    static Team besiktas = new Team(3, "besiktas", "BJ");
+    static Team sivasspor = new Team(4, "sivasspor", "SS");
+    static Team trabzonspor = new Team(5, "trabzonspor", "TS");
+    static Team göztepe = new Team(6, "göztepe", "GT");
+    static Team antalyaspor = new Team(7, "antalyaspor", "AS");
+    static Team gaziantepspor = new Team(8, "gaziantepspor", "GA");
+    static Team genclerbirligi = new Team(9, "genclerbirligi", "GB");
+    static Team bursaspor = new Team(10, "bursaspor", "BS");
+
+    public static void main(String[] args) {
+
+        TeamArray teamArr = new TeamArray();
+        MatchArray matchArr = new MatchArray();
+        Test t = new Test();
+
+        fillTeamArr(teamArr);
+        matchArr.fillMatchArr();
+        t.points(teamArr, matchArr);
+        teamArr.displayTeams();
+        matchArr.displayMatchArr();
+        teamArr.displayPoints();
+        champ(teamArr, matchArr);
+
+    }
+
+    public static void fillTeamArr(TeamArray teamArray) {
+        teamArray.teamArr[0] = galatasaray;
+        teamArray.teamArr[1] = fenerbahce;
+        teamArray.teamArr[2] = besiktas;
+        teamArray.teamArr[3] = sivasspor;
+        teamArray.teamArr[4] = trabzonspor;
+        teamArray.teamArr[5] = göztepe;
+        teamArray.teamArr[6] = antalyaspor;
+        teamArray.teamArr[7] = gaziantepspor;
+        teamArray.teamArr[8] = genclerbirligi;
+        teamArray.teamArr[9] = bursaspor;
+    }
+
+    public void points(TeamArray teamObj, MatchArray matchObj) {
```

Comment:
fonksiyon isimlendirmesi zayıf. calculatePoints daha anlaşılabilir olabilir.
lostMatch fonksiyonu boş olduğu için kullanmamıza gerek yok.
teamObj, matchObj anlaşılabilir parametre isimleri değil.

teams ve matchResults global değişkenleri oluşturduğumuz taktirde fonksiyona parametre bile göndermemize gerek kalmaz. örnek güncellenmiş fonksiyon;

```
    public void calculatePoints() {
        // Takımların puanlarını hesaplama
        for (int i = 0; i < NUMBER_OF_TEAMS; i++) {
            for (int j = 0; j < NUMBER_OF_TEAMS; j++) {
                if (i != j) {
                    int homeGoals = matchResults[i][j];
                    int awayGoals = matchResults[j][i];
                    if (homeGoals > awayGoals) {
                        teams[i].wonMatch();
                    } else if (homeGoals < awayGoals) {
                        teams[j].wonMatch();
                    } else {
                        teams[i].tie();
                        teams[j].tie();
                    }
                }
            }
        }
    }
```